src/features/lists/components/ListEditDistribution.tsx

```diff
@@ -32,6 +32,7 @@ import { AllocationForm } from "~/features/ballot/components/AllocationList";
 import { BallotSchema, type Vote } from "~/features/ballot/types";
```

Comment:
Is this needed? I thought we already removed the Lists feature?

I suppose it doesn't hurt 🤷

Reply:
I changed it for all the files to keep it consistent.